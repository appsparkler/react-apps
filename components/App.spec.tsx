import { render } from "@testing-library/react";

const App = () => null;

describe("App Snapshots", () => {
  it("should match snapshot", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
