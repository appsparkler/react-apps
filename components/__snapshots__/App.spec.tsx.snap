// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshots should match snapshot 1`] = `<div />`;
